exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,303 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "For this exercise it is possible to implement concurrency tests",
+    "but for language tracks that do not support concurrency, the tests",
+    "are written to be run in a single thread.",
+    "But for language tracks that do support concurrency, it is possible",
```

Comment:
This `But` doesn't work because the previous case also has a `but`. I suggest we change the entire thing to:

```
This exercise is a good candidate to practice concurrency, which
may not be available or possible in the implementing language track.
In this case, all tests can be executed on a single thread. If there
are tests that don't work using a single thread, these should be
marked with a concurrency scenario.
```

We probably also want to add `concurrency` to https://github.com/exercism/problem-specifications/blob/main/SCENARIOS.txt.